tests/Unit/Evolution/DgSubcell/Test_TwoMeshRdmpTci.cpp

```diff
@@ -35,7 +35,7 @@ struct Vector : db::SimpleTag {
 
 template <size_t Dim>
 DataVector soln(const tnsr::I<DataVector, Dim, Frame::Logical>& coords,
-                const size_t add_discontinuity_at) noexcept {
+                const bool add_discontinuity_at) noexcept {
```

Comment:
I suggest `add_discontinuity_at` -> `add_discontinuity`, because the "at" part doesn't make much sense now this it a bool. Sorry I missed this on my first pass.